pages/Community/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
Not something I'm going to not approve any commits for but there's a .pretterrc file all set up that should take care of code styling for us. Just noticed there was a semi colon here.